runtime/onert/core/src/compiler/CompilerFactory.cc

```diff
@@ -16,6 +16,8 @@
 
 #include "compiler/CompilerFactory.h"
 
+#include "MultiModelCompiler.h"
```

Comment:
(optional) Pull down this inclusion after "compiler/Compiler.h". ( It is not mandantory to sort the header. )